recipes/epysurv/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "epysurv" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jarnorfb/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 5dfd6e99b444cde80fe026d29a16428e15ea6208da0edcff0cb236f03296e4bc
+
+build:
+  number: 0
+  merge_build_host: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+  rpaths:
+    - lib/R/lib/
+    - lib/
```

Comment:
I don't think you need the `rpaths:` section. This is required for building R packages.

Reply:
True, now that I do not have to build surveillance myself, it can be removed.